_data/meetups.yml

```
@@ -1437,6 +1437,13 @@
   end_time: 13:00:00 EAT
   url: https://www.meetup.com/african_ruby_community/events/300548245
 
+- name: London Ruby User Group: September 2024 Meeting
```

Hey @fcheung, thanks for the PR!

If the value for a YAML column contains a `:` it sadly doesn't work. You either have to remove it, or wrap it in quotes. But if you want to match the style of the others I'd go with the dash:

```suggestion
- name: London Ruby User Group - September 2024 Meeting
```
or
```suggestion
- name: "London Ruby User Group: September 2024 Meeting"
```